File: src/main/java/serendustry/machine/RecipeMapTranscendentPlasmaMixer.java
package serendustry.machine;

import gregtech.api.recipes.RecipeMap;
import gregtech.api.recipes.builders.SimpleRecipeBuilder;

public class RecipeMapTranscendentPlasmaMixer extends
                                              RecipeMap<SimpleRecipeBuilder> {

    public RecipeMapTranscendentPlasmaMixer(String name, int inputs, int outputs, int fluidInputs, int fluidOutputs,
                                            boolean hidden) {
        super(name, inputs, outputs, fluidInputs, fluidOutputs, new SimpleRecipeBuilder(), hidden);
    }
    // todo custom page
}

File: src/main/kotlin/serendustry/machine/RecipeMapTranscendentPlasmaMixer.kt
package serendustry.machine

import gregtech.api.recipes.RecipeMap
import gregtech.api.recipes.builders.SimpleRecipeBuilder

class RecipeMapTranscendentPlasmaMixer(name: String, inputs: Int, outputs: Int, fluidInputs: Int, fluidOutputs: Int, hidden: Boolean):
    RecipeMap<SimpleRecipeBuilder>(name, inputs, outputs, fluidInputs, fluidOutputs, SimpleRecipeBuilder(), hidden) {

        // todo custom page
}
